_posts/0000-01-02-harunaltunhr.md

```diff
@@ -0,0 +1,5 @@
+layout: slide
+title: "Hello Everyone!"
+---
+Your text
```

Comment:
layout: slide
title: "Hello Everyone!"
---
Your text
Use the left arrow to go back!